<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreLancement</class>
 <widget class="QDialog" name="FenetreLancement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration du Match</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_17">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Paramétre du Match</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Nombre de set</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Score à atteindre</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Nombre de joueur</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Libéro</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="3">
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>Gestion du Match </string>
        </property>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>1er Arbitre</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditArbitre"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboBox_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>2éme Arbitre</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditArbitre_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBox_6">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_7">
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBox_8">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="checkBox_11">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Oui</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Score Initial</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Local</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Visiteur</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSpinBox" name="spinBox_9">
               <property name="maximum">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_10">
               <property name="maximum">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Gestion Equipe</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QComboBox" name="comboBox">
               <property name="minimumSize">
                <size>
                 <width>111</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>VS</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
          <zorder></zorder>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>label_11</zorder>
     <zorder>label_12</zorder>
     <zorder>label_13</zorder>
     <zorder>label_14</zorder>
     <zorder>groupBox_8</zorder>
     <zorder>spinBox_6</zorder>
     <zorder>spinBox_7</zorder>
     <zorder>spinBox_8</zorder>
     <zorder>checkBox_11</zorder>
     <zorder>groupBox_5</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>groupBox</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>Actions Analysés</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>Passe</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Contre</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Service</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Réception </string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Attaque</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>Défense</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="checkBox_18">
          <property name="text">
           <string>Relance</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Actions à defiler</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_16">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Service</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_14">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Contre</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_15">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Réception </string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_13">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Défense</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_12">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Attaque</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_17">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Passe</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Formule Statistique</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox_10">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Perso</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_8">
        <property name="text">
         <string>ValeurPP/Total</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_9">
        <property name="text">
         <string>(ValeurPP-ValeurMM)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(ValeurPP-ValeurMM)/Total</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QSpinBox" name="spinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>VPP +</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>VP +</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QSpinBox" name="spinBox_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>V0 -</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QSpinBox" name="spinBox_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLabel" name="label_5">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>VM -</string>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QSpinBox" name="spinBox_5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QLabel" name="label_6">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>VMM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="Line" name="line">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_7">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Total</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_annuler">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_4</zorder>
  <zorder></zorder>
  <zorder>groupBox_7</zorder>
  <zorder>groupBox_6</zorder>
 </widget>
 <tabstops>
  <tabstop>spinBox_6</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>spinBox_8</tabstop>
  <tabstop>checkBox_11</tabstop>
  <tabstop>spinBox_9</tabstop>
  <tabstop>spinBox_10</tabstop>
  <tabstop>lineEditArbitre</tabstop>
  <tabstop>lineEditArbitre_2</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>checkBox_8</tabstop>
  <tabstop>checkBox_9</tabstop>
  <tabstop>checkBox_10</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_annuler</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
